eng/config/PublishData.json

```diff
@@ -51,7 +51,7 @@
       "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
       "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
       "channels": [ "dev16.4p1" ],
-      "vsBranch": "master",
+      "vsBranch": "rel/d16.4",
```

Comment:
this isn't really the branch for dev16.4-p1 any more, but I couldn't find the branch and it isn't really going to matter until such time that we need to service preview 1.